File: 0001 Code Chef/0023 Chef Plays Ludo/implementation/0005 kotlin/Ludo.kt
fun main(args:Array<String>){
    var stringInput:String? = readLine();
    if(stringInput != null){
        var test:Int? = stringInput.toIntOrNull();
        if(test != null){
            var count:Int = test;
            while(count > 0){
                var stringInput:String? = readLine();
                if(stringInput != null){
                    var x:Int? = stringInput.toIntOrNull();
                    if(x != null){
                        if(x == 6){
                            println("YES");
                        }else{
                            println("NO");
                        }
                    }
                }
                count = count - 1;
            }
        }
    }
}

File: 0001 Code Chef/0023 Chef Plays Ludo/implementation/0009 dart/ludo.dart
import 'dart:io';

void main(){
  String? stringInput = stdin.readLineSync();
  if(stringInput != null){
    int? test = int.tryParse(stringInput);
    if(test != null){
      int count = test;
      while(count > 0){
        String? stringInput = stdin.readLineSync();
        if(stringInput != null){
          int? x = int.tryParse(stringInput);
          if(x != null){
            if(x == 6){
              print("YES");
            }else{
              print("NO");
            }
          }
        }
        count = count - 1; 
      }
    }
  }
}